storage/persistent_storage/rosa-persistent-storage-aws-efs-csi.adoc

```diff
@@ -0,0 +1,83 @@
+:_content-type: ASSEMBLY
+[id="rosa-persistent-storage-aws-efs-csi"]
+= Setting up AWS Elastic File Service CSI Driver Operator
+include::_attributes//attributes-openshift-dedicated.adoc[]
+:context: rosa-persistent-storage-aws-efs-csi
+toc::[]
+
+//Content similar to persistent-storage-csi-aws-efs.adoc and osd-persistent-storage-aws-efs-csi.adoc. Modules are reused.
+
+[IMPORTANT]
+====
+This procedure is specific to the Amazon Web Services Elastic File System (AWS EFS) CSI Driver Operator, which is only applicable for {product-title} 4.10 and later versions.
+====
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic File Service (EFS).
+
+Familiarity with link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html-single/storage/index#persistent-storage-overview_understanding-persistent-storage[persistent storage] and link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html-single/storage/index#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+After installing the AWS EFS CSI Driver Operator, {product-title} installs the AWS EFS CSI Operator and the AWS EFS CSI driver by default in the `openshift-cluster-csi-drivers` namespace. This allows the AWS EFS CSI Driver Operator to create CSI-provisioned PVs that mount to AWS EFS assets.
+
+* The _AWS EFS CSI Driver Operator_, after being installed, does not create a storage class by default to use to create persistent volume claims (PVCs). However, you can manually create the AWS EFS `StorageClass`.
+The AWS EFS CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
```

Comment:
Suggestion to break this sentence into two>> 
The AWS EFS CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand. 
This eliminates the need for cluster administrators to pre-provision storage.